src/pay/batch_payer.py

```diff
@@ -160,19 +166,31 @@ def pay(self, payment_items_in, dry_run=None):
             if pi.needs_activation and self.delegator_pays_ra_fee:
                 # Need to apply this fee to only those which need reactivation
                 zt += RA_BURN_FEE
-                sum_burn_fees += RA_BURN_FEE
 
+                # Check here if payout amount is greater than, or equal to new zero threshold with reactivation fee added.
+                # If so, add burn fee to global total. If not, payout will not get appended to list and therefor burn fee should not be added to global total.
+                if pi.amount >= zt:
```

Comment:
what is zt and pi? It would be good to spell out every variable for me who is new to the code base it's hard to read.